modules/installation-gcp-permissions.adoc

```diff
@@ -23,32 +24,64 @@ install {product-title}. To deploy an {product-title} cluster, the service
 account requires the following permissions. If you deploy your cluster into an existing VPC, the service account does not require certain networking permissions, which are noted in the following lists:
 
 .Required roles for the installation program
-* Compute Admin
-* Security Admin
-* Service Account Admin
-* Service Account User
-* Storage Admin
+[cols="3,7"options="header"]
+|===
+|Role|Description
+
+|Compute Admin
+|Gives full control of all compute resources. This role is needed by the service account to manage virtual machine instances. {product-title} is a self-managed platform that requires permission to create cloud infrastructure to automatically scale and failover during normal operation.
+
+|Security Admin
+|Allow the service account to create, modify, and delete firewall rules and SSL certificates within the GCP project.
+
+|Service Account Admin
+|Creates and manages service accounts.
```

Comment:
This is used to mint credentials. I'm not sure who the right team to reach out for to get clarification, though.

Reply:
It -might- not be needed when pass-through credentials are used.

Reply:
@joelddiaz do you have any insight into how this GCP permission affects the credentials workflow?

Reply:
The ability to create Service Accounts is the definition of "mint" mode. Passthrough mode would not need this as the creds stored in the Secret kube-system/gcp-credentials are just copied around to other Secrets in the passthrough case.